certbot/certbot/_internal/main.py

```diff
@@ -1483,7 +1485,7 @@ def make_displayer(config: configuration.NamespaceConfig
 
     :param config: Configuration object
 
-    :returns: Display object implementing :class:`certbot.interfaces.IDisplay`
+    :returns: Display object implementing :class:`certbot.interfaces.Display`
```

Comment:
There is no `certbot.interfaces.Display` object.

Reply:
I removed the reference.